ClosedXML_Tests/Excel/ConditionalFormats/ConditionalFormatCopyTests.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Linq;
+using ClosedXML.Excel;
+using NUnit.Framework;
+
+namespace ClosedXML_Tests.Excel.ConditionalFormats
+{
+    [TestFixture]
+    public class ConditionalFormatCopyTests
+    {
+        [Test]
+        public void StylesAreCreatedDuringCopy()
+        {
+            var wb = new XLWorkbook();
+            XLWorksheet ws = (XLWorksheet)wb.Worksheets.Add("Sheet");
```

Comment:
No need to cast to `XLWorksheet`. Please keep it simple, e.g. `var ws = wb.Worksheets.Add("Sheet");`